packages/e2e-tests/run-e2e.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+cd ../devtools
+yarn start:shared-ganache & 
+ganache=$!
+
+cd ../$1 && yarn run wait-on ../../.ganache-deployments/ganache-deployments-8545.json
```

Comment:
Previously, we waited on the *folder*, not on the file. The file is cleaned up when ganache winds down, but the folder is not. If starting from a clean hard drive (i.e. on circle) this is not a problem. On your local machine, it meant that this command would not wait properly after the first time you run it.

Reply:
Wouldn't the filename change if `GANACHE_PORT` was changed?

Reply:
Yes it would. But because of our `SC_ENV` solution, not easy to get the actual `GANACHE_PORT` into this script. 

It's the same problem to `wait-on localhost:3000`.